frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -362,17 +368,15 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                         throw new Error('Missing key')
                     }
 
-                    const blobResponseType = source.source === SnapshotSourceType.blob
-
-                    const response = blobResponseType
-                        ? await api.recordings.getBlobSnapshots(props.sessionRecordingId, params).catch((e) => {
-                              if (source.source === 'realtime' && e.status === 404) {
-                                  // Realtime source is not always available so a 404 is expected
-                                  return []
-                              }
-                              throw e
-                          })
-                        : (await api.recordings.listSnapshots(props.sessionRecordingId, params)).snapshots ?? []
+                    const response = await api.recordings
+                        .getBlobSnapshots(props.sessionRecordingId, params)
```

Comment:
Maybe we should rename this. It confused me that we fetch from the `realtime` source in this method. Maybe just `getSnapshots`